src/diffusers/hub_utils.py

```diff
@@ -72,6 +75,27 @@ def http_user_agent(user_agent: Union[Dict, str, None] = None) -> str:
     return ua
 
 
+def send_telemetry(data: Dict, name: str):
+    """
+    Sends logs to the Hub telemetry endpoint.
+
+    Args:
+        data: the fields to track, e.g. {"example_name": "dreambooth"}
+        name: a unique name to differentiate the telemetry logs, e.g. "diffusers_examples" or "diffusers_notebooks"
+    """
+    if DISABLE_TELEMETRY or HF_HUB_OFFLINE:
+        pass
```

Comment:
The check for `DISABLE_TELEMETRY` happened further down the stack originally, but let's indeed make it more explicit

Reply:
You'll want to `return` here, not `pass`, otherwise the telemetry will be sent regardless @anton-l

Reply:
Ah, seems like this was reported in an issue already, sorry for the duplicate.